### DISCUSSION TITLE
Reverb problem

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1728] [General]
Hello everyone, I created my organ from different samples mainly from SP, on Goodf. 

The organ works well on GO but when I play a piece on the Grand chœur (Toccata) sometimes the sound loses all reverberation and becomes dull for a few seconds and then everything returns to normal. My final notes always have a magnificent reverberation... is it a ram problem? I have 64GB .. or another problem internal to GO?

Many thanks !

**Comment by rousseldenis** [comment]
> Hello everyone, I created my organ from different samples mainly from SP, on Goodf.
> 
> The organ works well on GO but when I play a piece on the Grand chœur (Toccata) sometimes the sound loses all reverberation and becomes dull for a few seconds and then everything returns to normal. My final notes always have a magnificent reverberation... is it a ram problem? I have 64GB .. or another problem internal to GO?
> 
> Many thanks !

I had this problem too before. I had to increase polyphony to solve the problem(increase it step by step)

**Comment by oleg68** [comment]
I agree with @rousseldenis . Seems your polyphony level is not sufficient.

**Reply by Jean-Gabriel1982** [reply]
Polyphony is the number on main toolbar near the keyboard icon ? The number is 6.800 ..

**Reply by rousseldenis** [reply]
Yes. I've increased to 12000 with sample sets such as Alessandria on some opus that make every finger working :-)

If you are on Linux, an htop command while playing will show you CPUs intensive use that highlights that problem.

**Reply by rousseldenis** [reply]
I will add this setting should not be changed between sample sets but set once if that fits your hardware.

**Reply by Jean-Gabriel1982** [reply]
Many thanks for your answer !
Yes I'm on Linux Mint 21.2 ..
How I can do the htop ?

I'll try another day as I can't play with tutti at this time .. I don't know if my neighbours like the organ lol 😀
